.github/workflows/publish-build-layer-images.yml

```diff
@@ -0,0 +1,206 @@
+name: Publish build layer images
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_tag:
+        description: "Docker tag to be published"
+
+permissions:
+  packages: write
+  pull-requests: write
+  id-token: write
+  contents: read
+
+jobs:
+  build_rust_amd64:
+    uses: radixdlt/public-iac-resuable-artifacts/.github/workflows/docker-build.yml@main
+    with:
+      runs_on: ubuntu-16-cores-selfhosted
+      environment: "release"
+      image_registry: "docker.io"
+      image_organization: "radixdlt"
+      image_name: "babylon-node-build-layers"
+      tag: ${{ inputs.docker_tag }}-rust
+      context: "."
+      dockerfile: "Dockerfile"
```

Comment:
Hmm ... shouldn't this be `docker/dockerfiles/rust.dockerfile` etc? (or after rename, `docker/base-images/app.dockerfile`?)